raiden/constants.py

```diff
@@ -33,6 +33,46 @@
 SECONDS_PER_DAY = 24 * 60 * 60
 
 GENESIS_BLOCK_NUMBER = BlockNumber(0)
+
+# Relevant forks:
+# BYZANTIUM https://eips.ethereum.org/EIPS/eip-609
+# CONSTANTINOPLE https://eips.ethereum.org/EIPS/eip-1013
+
+
+class EthereumForks(Enum):
+    BYZANTIUM = BlockNumber(4_370_000)
+    CONSTANTINOPLE = BlockNumber(7_280_000)
+
+
+class RopstenForks(Enum):
+    BYZANTIUM = BlockNumber(1_700_000)
+    CONSTANTINOPLE = BlockNumber(4_230_000)
+
+
+class KovanForks(Enum):
+    BYZANTIUM = BlockNumber(0)
+    CONSTANTINOPLE = BlockNumber(4_230_000)
+
+
+class RinkebyForks(Enum):
+    BYZANTIUM = BlockNumber(0)
+    CONSTANTINOPLE = BlockNumber(3_660_663)
+
+
+class GoerliForks(Enum):
+    BYZANTIUM = BlockNumber(0)
+    CONSTANTINOPLE = BlockNumber(0)
+
+
+class Networks(Enum):
```

Comment:
This should be removed once https://github.com/raiden-network/raiden-contracts/issues/1226 is implemented.